global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "6.0.100",
+        "version": "6.0.400",
```

Comment:
Are you sure it shouldn't be ` 6.0.403`?

Reply:
Not sure, but here are some facts.

PolySharp 1.0.1 was released 2022-10-22, 6.0.403 was released on 2022-11-08.

https://github.com/Sergio0694/PolySharp/blob/main/src/PolySharp.SourceGenerators/PolySharp.targets#L53 mentions VS 17.3
VS 17.3 was released with 6.0.400

https://dotnet.microsoft.com/en-us/download/dotnet/6.0
![image](https://user-images.githubusercontent.com/919634/202906434-18731f74-b740-4948-b953-401f9053586e.png)

Reply:
I just tried out locally fixing to 6.0.302 and 6.0.400.
6.0.302 didn't work.
6.0.400 worked.